mars/tensor/special/airy.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2021 Alibaba Group Holding Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import scipy.special as spspecial
+
+from ..utils import infer_dtype, implement_scipy
+from .core import TensorTupleOp, _register_special_op
+
+
+@_register_special_op
+class TensorAiry(TensorTupleOp):
+    _func_name = "airy"
+    _n_outputs = 2
```

Comment:
Actual value of `n_ouptuts` = 4, but this is done to pass the `test_unary_tuple_operand` test in `test_special.py`. The same is done for `airye` and `itairy` functions.

Reply:
I updated the test function so the out tuple's size can be adjusted based on the operand. So using 4 for n_outputs should be fine now.